### PR TITLE
Update spdlog to last version

### PR DESCRIPTION
**Pull request by leoddias**
This workaround is for issue #702 , installing the last version of spdlog from git repository.

### CODE REVIEW

File: docker/server-only/build-env/Dockerfile.build [hunk 1]
```diff
@@ -12,7 +12,11 @@ RUN apk add --no-cache --repository http://dl-3.alpinelinux.org/alpine/edge/test
   make \
   mariadb-connector-c-dev \
   pugixml-dev \
-  spdlog-dev
+  git
+
+RUN git clone https://github.com/LIU-Yinyi/spdlog.git /tmp/spdlog \
```
Comment (Costallat) [inline docker/server-only/build-env/Dockerfile.build]: It's better to use the official repository

https://github.com/gabime/spdlog

Comment (leoddias) [reply docker/server-only/build-env/Dockerfile.build]: Well done. 💃

**Comment by Costallat** [comment]
Tested and its working now!

Thank you for your cooperation!

**Comment by majestyotbr** [comment]
Thanks for the contribution.

**Pull request merged**